packages/clay-tabs/src/index.tsx

```diff
@@ -10,38 +10,35 @@ import Content from './Content';
 import Item from './Item';
 import TabPane from './TabPane';
 
+export type DisplayType = null | 'tabs' | 'underline';
+
 interface IProps extends React.HTMLAttributes<HTMLUListElement> {
 	/**
-	 * Justify the nav items according the tab content.
+	 * Determines how to is displayed.
 	 */
-	justified?: boolean;
+	displayType?: DisplayType;
 
 	/**
-	 * Applies a modern style to the tab.
+	 * Justify the nav items according the tab content.
 	 */
-	modern?: boolean;
+	justified?: boolean;
 }
 
 const ClayTabs: React.FunctionComponent<IProps> & {
 	Content: typeof Content;
 	TabPane: typeof TabPane;
 	TabPanel: typeof TabPane;
 	Item: typeof Item;
-} = ({
-	children,
-	className,
-	justified,
-	modern = true,
-	...otherProps
-}: IProps) => {
+} = ({children, className, displayType, justified, ...otherProps}: IProps) => {
+	const defaultValue = displayType === null;
+
 	return (
 		<ul
 			className={classNames(
 				'nav',
 				{
+					[`nav-${displayType}`]: !defaultValue,
```

Comment:
Since we're just using it here, I think we can avoid creating a variable for this and adding the conditional directly here just to avoid allocating a new variable.